README.md

```diff
@@ -24,8 +24,8 @@
 - [Python 3.9](https://www.python.org/downloads/release/python-390/)
 - [Git](https://www.atlassian.com/git/tutorials/install-git)
 - [Blender](https://www.blender.org/download/)
-- At least 2 simple webcameras (more cameras, better quality)
-    - [$10 camera here](https://www.amazon.com/1080P-Full-Webcam-Microphone-Conferencing/dp/B09JFPFVY4/ref=asc_df_B09JFPFVY4/?tag=hyprod-20&linkCode=df0&hvadid=563548498710&hvpos=&hvnetw=g&hvrand=17033983634274965573&hvpone=&hvptwo=&hvqmt=&hvdev=c&hvdvcmdl=&hvlocint=&hvlocphy=9028321&hvtargid=pla-1629009316502&psc=1)
```

Comment:
replacing the link with a camera that I'm confident will work.

Some of those SUPER cheap cameras (in the $10 range) don't work for some unknown reason.

I have a pile of those kinds of cameras in my apartment, I can check if they work with the new code

Reply:
we should get a working camera list then. future thing